src/components/Header/Header.js

```diff
@@ -29,7 +48,30 @@ export const Header = () => {
         </header>
         <SideMenu />
       </div>
-  )
+    )
+  }
+}
+
+const mapStateToProps = (state) => {
+	    return {
+        cropImageName: state.cropImageName,
+        cycleFlag: state.cycleFlag
+        }
+}
+
+const cycleThrough = (cropImageName) => {
+	return {type: "CHANGE_CROP_IMAGE", payload: cropImageName }

```

Comment:
Code styling suggestion. It's good practice to not use magic strings, but instead to use constant variable or some other key that is not near as easy to typo. Your IDE will often autocomplete a constant for you or linting may detect that the variable does not exist if you typo it. If you use a magic string there's a larger chance of it slipping past with a typo.

Also I realized that linting is disabled by default on build in the new version of the starter kit. What kind of monsters do that?